src/components/marketing/ContactUsForm.spec.js

```diff
@@ -0,0 +1,62 @@
+import ContactUsForm from "./ContactUsForm";
+import React from "react";
+import { render, fireEvent, cleanup } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { BrowserRouter as Router } from "react-router-dom";
+
+afterAll(cleanup);
+
+describe("Render testing", () => {
+  it("Full Name form test", () => {
+    const { getByLabelText } = render(
+      <Router>
+        <ContactUsForm />
+      </Router>
+    );
+    const fullNameInput = getByLabelText(/Full Name*/i);
+    expect(fullNameInput).toBeVisible();
+  });
+  it("Email form test", () => {
+    const { getByLabelText } = render(
+      <Router>
+        <ContactUsForm />
+      </Router>
+    );
+    const emailInput = getByLabelText(/Email*/i);
+    expect(emailInput).toBeVisible();
+  });
+  it("Message form test", () => {
+    const { getByText } = render(
+      <Router>
+        <ContactUsForm />
+      </Router>
+    );
+    const messageInput = getByText(/Message*/i);
+    expect(messageInput).toBeVisible();
+  });
+});
+
+describe("functionality test", () => {
```

Comment:
Similarly, this would be better to have the name of the tested component here, like "Contact Form functionality". For future reference, the whole "describe" and "it" design is meant to sound almost like a sentence